frontend/public/components/ingress.jsx

```diff
@@ -199,47 +177,86 @@ const RulesRows = (props) => {
   return <EmptyBox label="Rules" />;
 };
 
-const Details = ({ obj: ingress }) => (
-  <>
-    <div className="co-m-pane__body">
-      <SectionHeading text="Ingress Details" />
-      <ResourceSummary resource={ingress}>
-        <dt>TLS Certificate</dt>
-        <dd>{getTLSCert(ingress)}</dd>
-      </ResourceSummary>
-    </div>
-    <div className="co-m-pane__body">
-      <SectionHeading text="Ingress Rules" />
-      <p className="co-m-pane__explanation">
-        These rules are handled by a routing layer (Ingress Controller) which is updated as the
-        rules are modified. The Ingress controller implementation defines how headers and other
-        metadata are forwarded or manipulated.
-      </p>
-      <div className="co-m-table-grid co-m-table-grid--bordered">
-        <RulesHeader />
-        <RulesRows spec={ingress.spec} namespace={ingress.metadata.namespace} />
+const Details = ({ obj: ingress }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('ingress~Ingress details')} />
+        <ResourceSummary resource={ingress}>
+          <dt data-test="tls-certificate">{t('ingress~TLS certificate')}</dt>
+          <dd>
+            <TLSCert ingress={ingress} />
+          </dd>
+        </ResourceSummary>
       </div>
-    </div>
-  </>
-);
-
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('ingress~Ingress rules')} />
+        <p data-test="ingress-rules-desc" className="co-m-pane__explanation">
+          {t(
+            'ingress~These rules are handled by a routing layer (Ingress Controller) which is updated as the rules are modified. The Ingress controller implementation defines how headers and other metadata are forwarded or manipulated',
```

Comment:
Ingress controller capitalization is inconsistent here - I'm not sure what's right.

Reply:
Hi @rebeccaalpert, these capitilization issues should be filed in #6928!  -thanks